apps/frontend/src/assets/css/scaffolding.css

```diff
@@ -8,6 +10,7 @@ body {
 	min-width: 320px;
 	min-height: 100vh;
 	margin: 0;
+	overflow: hidden;
```

Comment:
Try not to change global styles

Reply:
sorry about that, i remove this line now